src/test/java/uk/gov/hmcts/divorce/divorcecase/NoFaultDivorceTest.java

```diff
@@ -19,12 +19,12 @@ public class NoFaultDivorceTest {
     private NoFaultDivorce noFaultDivorce;
 
     @Test
-    void shouldAddSystemUpdateUserAccessToDraftStateWhenEnvironmentIsAat() {
+    void shouldSetTheCorrectCaseTypeName() {
         final ConfigBuilderImpl<CaseData, State, UserRole> configBuilder = createCaseDataConfigBuilder();
 
         noFaultDivorce.configure(configBuilder);
 
-        assertThat(configBuilder.build().getCaseType()).isEqualTo("NFD");
+        assertThat(configBuilder.build().getCaseDesc()).isEqualTo("Handling of the dissolution of marriage");
```

Comment:
test says correct case type but this is comparing case description. is that correct?

Reply:
this is a bit of a fudge as I wasn't able to construct a property test as System.getenv() is immutable